src/Models/WebMapServiceCatalogGroup.js

```diff
@@ -59,6 +59,12 @@ var WebMapServiceCatalogGroup = function(application) {
      */
     this.blacklist = undefined;
 
+    /**
+     * Specifies whether to use each WMS layer's "title" (default), "name", or "abstract" as the primary title.
+     * @type {String}
+     */
+    this.titlefield = 'title';
```

Comment:
Sorry for the nitpick, but can you make this camelCase: `titleField`?  Also, start the doc with "Gets or sets" for a read/write property like this, or just "Gets" if it's read-only.